### DISCUSSION TITLE
v1.1.8-fix1

### DISCUSSION THREAD

**Discussion by constanline** [constanline/XQuickEnergy#318] [Announcements/公告]
**Full Changelog**: https://github.com/constanline/XQuickEnergy/compare/v1.1.8...v1.1.8-fix1

<hr /><em>This discussion was created from the release <a href='https://github.com/constanline/XQuickEnergy/releases/tag/v1.1.8-fix1'>v1.1.8-fix1</a>.</em>

**Comment by xiaolanchong123** [comment]
这么速度的吗

**Comment by xzcgg** [comment]
加载成功显示异常，显示为res/xml/tab_text_color.xml加载成功

**Comment by WskDnpl** [comment]
希望这个hook成功能加个开关，每次打开手机都会弹出来加载成功

**Comment by alinnse** [comment]
> 希望这个hook成功能加个开关，每次打开手机都会弹出来加载成功

把 气泡提示 关了

**Reply by WskDnpl** [reply]
关了收能量就不会有提示了

**Comment by xiaojian1880** [comment]
有木有加好友功能
